xml/bp_chap_getting_started_with_salt_formulas.xml

```diff
@@ -436,12 +431,21 @@ timezone:
       field and the <literal>color</literal> type will generate a color picker.
      </para>
      <para>
-      The <literal>group</literal> and <literal>hidden-group</literal> types do
-      not generate an editable field and are used to structure form and pillar
-      data. The difference between group and hidden-group is
-      <literal>group</literal> generates a visible border with a heading, and
-      hidden-group shows nothing visually (and is only used to structure pillar
-      data).
+      The <literal>group</literal>, <literal>edit-group</literal>, and
+      <literal>namespace</literal> (formerly
+      <literal>hidden-group</literal>) types do not generate an editable
+      field and are used to structure form and pillar data. The
+      difference between group and namespace is <literal>group</literal>
+      generates a visible border with a heading, and namespace shows
+      nothing visually (and is only used to structure pillar data).  The
+      difference between group and edit-group is
+      <literal>edit-group</literal> allows to structure and restrict
+      editable fields in a more flexible way. For example,
+      <literal>edit-group</literal> supports the
+      <literal>$minItems</literal> and <literal>$maxItems</literal>
+      attributes, and thus it simplifies creating complex, but somehow
```

Comment:
Subjective opinion: maybe i'd stay with `....it simplifies complex and repeatable input structures.`